framer/Utils.coffee

```diff
@@ -434,6 +434,14 @@ Utils.deviceFont = (os) ->
 	return "Segoe UI" if os is "windows"
 	return "Helvetica"
 
+Utils.webFont = (font) ->
+	link = document.createElement("link")
+	link.href = "https://fonts.googleapis.com/css?family=#{font}"
+	link.rel = "stylesheet"
+	document.getElementsByTagName("head")[0].appendChild(link)
+
+	return font
```

Comment:
Hmm. I feel like `webFont` does appropriately illustrate that it is being 'loaded'. For your second point, what do you think of `Utils.googleWebFont` or `Utils.googleFont`? I dislike the longer names, so would prefer `webFont` personally, but I can see how it's not immediately clear. 

I see how that could cause errors. Would you add early returns to throw an error in case the input is invalid? Or, how else to solve this?